src/main/java/frc/robot/Constants.java

```diff
@@ -19,6 +19,9 @@
     //controller port
     public static final int controllerport = 0;
     //motor controller ports
+    public static final int elevatorPort = 4;
+    //other constants
+    public static final double elevatorSpeed = 0.6; //speed for elevator to move at
     public static final int leftmotor = 0;
     public static final int rightmotor = 1;
     public static final int outputMotorPort = 2;
```

Comment:
I don't know if this is your fault, but these should be under the motor controller ports, not other constants.